exonum/src/storage/indexes_metadata.rs

```diff
@@ -172,7 +172,23 @@ pub fn set_index_type(name: &str, index_type: IndexType, is_family: bool, view:
 #[cfg(test)]
 mod tests {
     use super::{IndexType, IndexMetadata, INDEXES_METADATA_TABLE_NAME};
-    use storage::{MemoryDB, Database, MapIndex, ProofMapIndex};
+    use storage::{MemoryDB, Database, MapIndex, ProofMapIndex, StorageKey};
+
+    impl StorageKey for [u8; 32] {
```

Comment:
Because without it, the metadata tests (and only metadata tests) don't work :upside_down_face: In the present implementation, `ProofMapKey` is completely delineated from `StorageKey` (imo, quite deservedly so - the user-supplied key types are *not* stored as keys, `ProofPath`s are), so, as a consequence, there is no more `StorageKey` implementation for `[u8; 32]` (it was previously defined [in `proof_map_index/keys.rs`](https://github.com/exonum/exonum/blob/8e01e2ec04ec92bea8012a4f4da1e389d8d7708c/exonum/src/storage/proof_map_index/key.rs#L42)). Obviously, there may be other solutions:

- Defining `StorageKey` for `[u8; 32]` in `storage/keys.rs`
- Changing `[u8; 32]` to some other type in metadata tests

I'm inclined to go with the second route. Would you be satisfied with it?

Reply:
Well, I think second option would be okay. I don't think we should define `StorageKey` for some types only for test purposes, it seems weird.